### PR TITLE
add test HE003_TC13,14,15

### PR DESCRIPTION
**Pull request by MelisPavol**

### CODE REVIEW

File: src/test/java/AppointmentTest.java [hunk 1]
```diff
@@ -287,16 +287,16 @@ void test_HE_003_TC_12_Valid_Login_From_Hamburger_Menu_Medial_Care_MediaCare() {
         assertEquals(getTextHealthMadicareCheckBox,"Medicare");
     }
     @Test
-    void test_HE_003_TC_13_Valid_Login_From_Hamburger_Menu_Medial_Care_MediaCare() {
-        var getTextHealthMadicareCheckBox = homePage
+    void test_HE_003_TC_13_Valid_Login_From_Hamburger_Menu_Medial_Care_Medicaid() {
+        var getTextHaelthMedicaidCheckBox = homePage
                 .hamburgerMenu()
                 .loginFromHamburgerMenu()
```
Comment (darinamas) [inline src/test/java/AppointmentTest.java]: The name of variable is incorrect "getTextHaelthMedicaidCheckBox"
Should be "TextHaelthMedicaidCheckBox"

File: src/test/java/AppointmentTest.java [hunk 2]
```diff
@@ -306,9 +306,55 @@ void test_HE_003_TC_13_Valid_Login_From_Hamburger_Menu_Medial_Care_MediaCare() {
...
                 .historyButton()
-                .getTextHealthMadicareCheckBox();
-        assertEquals(getTextHealthMadicareCheckBox,"Medicare");
+                .getTextHaelthMedicaidCheckBox();
+        assertEquals(getTextHaelthMedicaidCheckBox,"Medicaid");
     }
+    @Test
+    void test_HE_003_TC_14_Valid_Login_From_Hamburger_Menu_Medial_Care_None() {
+        var getTextHealthNoneCheckBox = homePage
+                .hamburgerMenu()
```
Comment (darinamas) [inline src/test/java/AppointmentTest.java]: Please rename variable "getTextHealthNoneCheckBox"

File: src/test/java/AppointmentTest.java [hunk 3]
```diff
@@ -306,9 +306,55 @@ void test_HE_003_TC_13_Valid_Login_From_Hamburger_Menu_Medial_Care_MediaCare() {
...
+                .clickHamburgerMenuSummaryPage()
+                .historyButton()
+                .getTextHealthNoneCheckBox();
+        assertEquals(getTextHealthNoneCheckBox,"None");
+    }
+    @Test
+    void test_HE_003_TC_15_Valid_Login_From_Hamburger_Menu_dont_write_comment() {
+        var getTextComment = homePage
+                .hamburgerMenu()
+                .loginFromHamburgerMenu()
```
Comment (darinamas) [inline src/test/java/AppointmentTest.java]: Please rename variable

**Review by darinamas — CHANGES_REQUESTED** [review]

File: src/test/java/AppointmentTest.java (hunk 2 above)
Comment (MelisPavol) [reply src/test/java/AppointmentTest.java]: fixed

File: src/test/java/AppointmentTest.java (hunk 3 above)
Comment (MelisPavol) [reply src/test/java/AppointmentTest.java]: fixed

File: src/test/java/AppointmentTest.java (hunk 1 above)
Comment (MelisPavol) [reply src/test/java/AppointmentTest.java]: fixed

File: src/test/java/AppointmentTest.java (hunk 2 above)
Comment (MelisPavol) [reply src/test/java/AppointmentTest.java]: fixed

File: src/test/java/AppointmentTest.java (hunk 3 above)
Comment (MelisPavol) [reply src/test/java/AppointmentTest.java]: fixed

**Pull request merged**